src/swc.rs

```diff
@@ -369,6 +382,24 @@ fn infer_name_from_expr(mut expr: &ast::Expr) -> Option<ScopeName> {
     }
 }
 
+fn computed_prop_name_to_component(prop: &ast::ComputedPropName) -> Option<(NameComponent, bool)> {
+    if let Some(lit) = prop.expr.as_lit() {
+        if let ast::Lit::Str(prop) = lit {
+            return Some((NameComponent::interp(prop.value.to_string()), true));
```

Comment:
a string is not necessarily a valid identifier, you could have `"all kinds of craaaaazy stuff"` in there ;-)
better to use `interp(format!("[{:?}]", prop.value))`. I believe we can just do that for all kinds of literals, as there could also be other types I believe?